From 6dc810804bb2fa91d9972601c209471fab852509 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 26 Aug 2022 20:50:51 +0200
Subject: [PATCH] fixup! net: sfp: add support for HALNy GPON SFP

There are a few typos, which was reported by Orest Worhacz
(@AreYouLoco), which needs to be fixed.

SFP is detected as
[    9.607535] sfp sfp: module HALNy HL-GSFP          rev V1.0 sn HALN1010493c     dc 20150525
---
 drivers/net/phy/sfp.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/phy/sfp.c b/drivers/net/phy/sfp.c
index 2ebc577bfc56..277292e2ed35 100644
--- a/drivers/net/phy/sfp.c
+++ b/drivers/net/phy/sfp.c
@@ -329,7 +329,7 @@ static void sfp_fixup_inverted_los(struct sfp *sfp)
 	sfp->id.ext.options |= los_inverted;
 }
 
-static void sfp_fixup_hanly_gsfp(struct sfp *sfp)
+static void sfp_fixup_halny_gsfp(struct sfp *sfp)
 {
 	/* LOS is inverted */
 	sfp_fixup_inverted_los(sfp);
@@ -369,9 +369,9 @@ static const struct sfp_quirk sfp_quirks[] = {
 		.modes = sfp_quirk_2500basex,
 		.fixup = sfp_fixup_long_startup,
 	}, {
-		.vendor = "HANLy",
+		.vendor = "HALNy",
 		.part = "HL-GSFP",
-		.fixup = sfp_fixup_hanly_gsfp,
+		.fixup = sfp_fixup_halny_gsfp,
 	}, {
 		// Huawei MA5671A can operate at 2500base-X, but report 1.2GBd
 		// NRZ in their EEPROM
-- 
2.34.1

